src/Operations.jl

```diff
@@ -129,6 +136,7 @@ function update_manifest!(ctx::Context, pkgs::Vector{PackageSpec}, deps_map)
             entry.deps = deps_map[pkg.uuid]
         end
         ctx.env.manifest[pkg.uuid] = entry
+        precomp_unsuspend!(pkg)
```

Comment:
@fredrikekre this is where it's unsuspended. If any change is made to that package in the manifest